### PR TITLE
Revert back to using out of the box default Edge settings

### PR DESCRIPTION
**Pull request by celonicolosi**
This is a bugfix of the previous bugfix. The previous bugfix was not working properly when running tests local with Edge browser on Mac.

**Bugfixes**
* Fix Load error in Edge Options
  * Mac Edge was not properly processing the browser name by default

### CODE REVIEW

**Comment by luke-hill** [comment]
So this was actually an issue with Edge beta on Mac, not Edge wholesale on Mac.

The reason we did this is because Edge **only** exists in beta for Linux (Don't know why)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
 ## <sub>main</sub>
 #### _Unreleased_
 
-## <sub>v3.0.1</sub>
+## <sub>v3.0.2</sub>
 #### _Unreleased_
+**Bugfixes**
+* Fixed load error in Selenium Webdriver Edge
```
Comment (luke-hill) [inline CHANGELOG.md:7]: * Fix Load error in Edge Options
  * Mac Edge was not properly processing the browser name by default

**Review by luke-hill — CHANGES_REQUESTED** [review]
Part of the explanation here is wrong. Mac can use beta, but we need to provide an additional property (Non-default), for this. Allowing people to provide additional properties to options e.t.c. is something we haven't yet allowed in our code, so for now let's just mention that Mac Edge had a slight bug.

**Review by luke-hill — APPROVED** [review]

File: lib/ca_testing/drivers/v4/options.rb [hunk 2]
```diff
@@ -30,22 +23,13 @@ def initial_options(browser)
...
             when :firefox;            then ::Selenium::WebDriver::Firefox::Options.new(log_level: "trace")
-            when :edge;               then edge_options
+            when :edge;               then ::Selenium::WebDriver::Edge::Options.new
             when :safari;             then ::Selenium::WebDriver::Safari::Options.new(automatic_inspection: true)
             when :internet_explorer;  then internet_explorer_options
             else {}
             end
           end
 
-          # This is only needed before version 4.0 is stable
```
Comment (luke-hill) [inline lib/ca_testing/drivers/v4/options.rb:40]: The EdgeHtml options (v3), is actually "more" stable than the EdgeChrome in v4 alpha.

EdgeChrome became the default in the rc1 release (But by then had some bug fixes), hence this plaster for all working perfectly.

**Review by luke-hill — APPROVED** [review]

**Review by david-lake — APPROVED** [review]

**Pull request merged**